egs/wsj/s5/steps/lmrescore_rnnlm_lat.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Copyright 2015  Guoguo Chen
+# Apache 2.0
+
+# This script rescores lattices with RNNLM.
+
+# Begin configuration section.
+cmd=run.pl
+skip_scoring=false
+max_ngram_order=4
+N=10
+inv_acwt=12
+weight=1.0  # Interpolation weight for RNNLM.
+# End configuration section.
+
+echo "$0 $@"  # Print the command line for logging
+
+. ./utils/parse_options.sh
+
+if [ $# != 5 ]; then
+   echo "Does language model rescoring of lattices (remove old LM, add new LM)"
+   echo "with RNNLM."
+   echo ""
+   echo "Usage: $0 [options] <rnnlm-dir> <old-lang-dir> \\"
+   echo "                   <data-dir> <input-decode-dir> <output-decode-dir>"
+   echo " e.g.: $0 rnnlm data/lang_tg data/test \\"
```

Comment:
I'd prefer (just for the sake of easy understanding to write the example line in this was
 $0 ./rnnlm data/lang_tg data/test

so that it would be completely obvious that rnnlm is a directory :)